### DISCUSSION TITLE
Open sqlite database?

### DISCUSSION THREAD

**Discussion by nRoger-Env** [Giorgi/DuckDB.NET#187] [Q&A] [answered]
DuckDB allows connecting directly to an Sqlite database with an Sqlite extension (see here: https://duckdb.org/docs/extensions/sqlite)
Is it possible to do the same when using DuckDB.NET?

**Comment by Giorgi** [comment]
Everything that you can do with "plain" DuckDB can be done from the .NET library too.

**Reply by nRoger-Env** [reply, marked as answer]
Ha you are right. I think I first tried it with a corrupted sqlite DB, which is why I couldn't make it work.
It turns out it is indeed very easy. Exemple with Dapper ORM on top:
```
using var duckLiteCon = new DuckDBConnection("Data Source=duckDB_Sqlite.sqlite3");
duckLiteCon.Open();
var dataDuckLite = duckLiteCon.Query("SELECT * FROM Table1");
foreach (var item in dataDuckLite)
{
    Console.WriteLine(item.ID);
}
duckLiteCon.Close();
```
Thanks
